workflow-core/src/commonMain/kotlin/com/squareup/workflow1/StatefulWorkflow.kt

```diff
@@ -1007,6 +1007,8 @@ public fun <PropsT, StateT, OutputT, RenderingT>
     name: () -> String,
     update: WorkflowAction<PropsT, StateT, OutputT>.Updater.() -> Unit
   ): WorkflowAction<PropsT, StateT, OutputT> = object : WorkflowAction<PropsT, StateT, OutputT>() {
+  override val debuggingName: String
```

Comment:
Should update the `@param` for `name` to mention `debuggingName` instead of `toString()`. Just to draw people's attention to the better thing.

Reply:
Done.